scripts/configure_pyproject_toml.py

```diff
@@ -51,7 +61,16 @@ def parse_args():
     parsed_args = parse_args()
     pyproject_path = os.path.join(parsed_args.path, "pyproject.toml")
 
-    pyproject = toml.load(pyproject_path)
+    if not has_toml:
+        raise ImportError(
+            "A TOML parser is required to configure 'pyproject.toml'. "
+            "We support any of the following TOML parsers: [tomli, tomlkit, tomllib] "
+            "You can install either tomli via `pip install tomli`, "
+            "tomlkit via `pip install tomlkit`, or use Python 3.11 "
+            "or above which natively offers the tomllib library."
+        )
+
+    pyproject = tomllib.load(pyproject_path)
```

Comment:
These other Python packages need a binary file object. Please update everywhere accordingly:
```suggestion
    with open(pyproject_path, "rb") as f:
        pyproject = tomllib.load(f)
```